testsuite/features/secondary/proxy_container_cobbler_pxeboot.feature

```diff
@@ -26,7 +26,8 @@ Feature: PXE boot a terminal with Cobbler and containerized proxy
   # The build host has same version as the terminal
   Scenario: Prepare the autoinstallation files on the server
     When I install packages "tftpboot-installation-SLE-15-SP4-x86_64 expect" on this "build_host"
-    And I copy the tftpboot installation files from the build host to the server
+    And I perform a key exchange to allow machine:"build_host" authenticate to machine:"server"
```

Comment:
If we have this need, we should introduce it as part of our IaC. I can guide  you to make this on Sumaform. Let's talk about it today.